consensus-types/blocks/execution.go

```diff
@@ -400,13 +410,14 @@ func PayloadToHeader(payload interfaces.ExecutionData) (*enginev1.ExecutionPaylo
 // This wrapper allows us to conform to a common interface so that beacon
 // blocks for future forks can also be applied across Prysm without issues.
 type executionPayloadCapella struct {
-	p     *enginev1.ExecutionPayloadCapella
-	value uint64
+	p         *enginev1.ExecutionPayloadCapella
+	weiValue  math.Wei
+	gweiValue uint64
```

Comment:
do we still need to keep `gweiValue` now we have `weiValue`?

Reply:
I kept `gweiValue` so that we don't have to convert every time we want to get the value in Gwei.